docs/products/postgresql/concepts/pg-connection-pooling.md

```diff
@@ -2,28 +2,39 @@
 title: Connection pooling
 ---
 
-Connection pooling in Aiven for PostgreSQL® services allows you to
-maintain very large numbers of connections to a database while
-minimizing the consumption of server resources.
+Connection pooling in Aiven for PostgreSQL® services allows you to maintain very large numbers of connections to a database while minimizing the consumption of server resources.
+
+## About connection pooling
 
 Aiven for PostgreSQL connection pooling uses
 [PgBouncer](https://www.pgbouncer.org/) to manage the database
 connection.
 
-:::note
```

Comment:
I believe we should keep this original key information, perhaps again in a Node card without the explanation text (as it is explained below). Otherwise two key info is split by the example. 

- 500 for each GB of RAM in the service plan
-  A total of 50000 client connections

Reply:
Instead of duplicating this info in two sections, I reorganized the other section so that the key info is no longer split by the example. Please have a look. Wdyt?

Reply:
Thanks. I think this version is much better as it retains the key information together, more visible but it also explains supported by an example. 👌🏻

The only remaining point for me is that 50.000 is the limit per service and there is no way of exceeding it. You might consider dropping the part "and applies every time your calculated connection number exceeds 50000."